pkg/kv/kvserver/replica_consistency_diff.go

```diff
@@ -36,46 +35,38 @@ type ReplicaSnapshotDiff struct {
 type ReplicaSnapshotDiffSlice []ReplicaSnapshotDiff
 
 // WriteTo writes a string representation of itself to the given writer.
-func (rsds ReplicaSnapshotDiffSlice) WriteTo(w io.Writer) (int64, error) {
-	n, err := w.Write([]byte("--- leaseholder\n+++ follower\n"))
-	if err != nil {
-		return 0, err
-	}
+func (rsds ReplicaSnapshotDiffSlice) WriteTo(buf *redact.StringBuilder) {
+	buf.Printf("--- leaseholder\n+++ follower\n")
 	for _, d := range rsds {
-		prefix := "+"
+		prefix := redact.Safe("+")
```

Comment:
`SafeString`

Reply:
Changed - I had that originally but the comment on `SafeString` seemed to indicate using it was somehow "worse" than using `Safe`.

Reply:
(no push yet)